core/ast/src/scope.rs

```diff
@@ -347,14 +375,17 @@ impl Scope {
             return;
         }
         let binding_index = bindings.len() as u32;
+        let mut flags = BindingFlags::LEX;
+        if strict {
+            flags |= BindingFlags::STRICT;
+        }
```

Comment:
We can simplify this if we use [`Flags::set()`](https://docs.rs/bitflags/latest/bitflags/trait.Flags.html#method.set) (applies to all)
```suggestion
        flags.set(BindingFlags::STRICT, strict);
```